OpenRA.Game/Map/MapCache.cs

```diff
@@ -174,7 +174,8 @@ void LoadAsyncInternal()
 					if (bitmap == null)
 					{
 						createdPreview = true;
-						bitmap = Minimap.RenderMapPreview(modData.DefaultRules.TileSets[p.Map.Tileset], p.Map, modData.DefaultRules, true);
+						var map = new Map(p.Path);
```

Comment:
This is going to be replaced with a much lighterweight parser in a future pr.  Is this bogus enough to have to expand the scope of the pr with these extra changes now?